torchinductor/decomposition.py

```diff
@@ -20,12 +20,14 @@
         aten._adaptive_avg_pool2d_backward,
         aten.addcmul,
         aten.avg_pool2d_backward,
+        aten.binary_cross_entropy_with_logits,
         aten.clamp_max,
         aten.clamp_min,
         aten.col2im_backward,
         aten.cudnn_batch_norm,
         aten.cudnn_batch_norm_backward,
         aten.detach,
+        aten.dot,
```

Comment:
but this hasn't landed yet?

Reply:
yeah but if it doesn't exist then it won't get it lol.

There's a bunch of decomps in here that don't exist :P